src/main/kotlin/org/elm/lang/core/psi/ElementTags.kt

```diff
@@ -11,10 +12,12 @@ interface ElmExpressionPartTag : ElmPsiElement
 /** An element that can occur in an [ElmExpression] as the argument to a function or operator */
 interface ElmOperandTag : ElmExpressionPartTag
 
-/** An element that can be the parameter of an [ElmFunctionDeclarationLeft], [ElmAnonymousFunction],
- * or [ElmCaseOfBranch] */
+/** An element that can be the parameter of an [ElmFunctionDeclarationLeft], [ElmAnonymousFunction], or [ElmCaseOfBranch] */
 interface ElmNameDeclarationPatternTag : ElmNamedElement
 
+/** A function being called as the child of a [ElmFunctionCall] */
+interface ElmFunctionCallTarget : ElmReferenceElement
```

Comment:
I don't see any classes getting tagged with this new interface.

Reply:
good catch, sorry about that